manifests/init.pp

```diff
@@ -140,4 +137,15 @@
   }
   ensure_resources('file', $files_absent , $files_absent_defaults)
 
+  # THERE IS A BUG IN PAM SUPPORT FOR motd.d FILES THAT IT DOESN'T SORT THEM
```

Comment:
This is the primary change. Curious what you think of this solution to the motd.d ordering issue.